.github/workflows/pr-workflow.yml

```diff
@@ -82,12 +84,18 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ steps.init.outputs.REGION }}
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
       - name: Run build script
         id: builder
         run: |
-          cd deployment
+          python3.9 --version
```

Comment:
Does this version print serve a purpose, or can it be removed?

Reply:
it can be removed, was testing to see the actual version of python 3.9 it was using from github action.